Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/helpers/UIHelper.java

```diff
@@ -74,6 +75,8 @@ void showException(@NotNull String message,
 
     void openWebAppPropertyView(@NotNull WebAppNode node);
 
+    void openDeploymentSlotPropertyView(@NotNull DeploymentSlotNode node);
```

Comment:
It fails Eclipse build if you doesn't implement the interface in that.

Reply:
Yes, miss this is the change list.